lib/modules/manager/npm/extract/types.ts

```diff
@@ -31,3 +31,5 @@ export interface LockFile {
 export interface PnpmWorkspaceFile {
   packages: string[];
 }
+
+export type OverrideDependency = Record<string, string | any>;
```

Comment:
please use a recursive type declaration like 
```suggestion
export type OverrideDependency = Record<string, string | OverrideDependency;
```